packages/@okta/vuepress-theme-prose/components/FrontPageWidget.vue

```diff
@@ -3,25 +3,24 @@
 </template>
 
 <script>
-// globals: OktaSignIn
-// OktaSignIn loaded from CDN script in <head>
+import '@okta/okta-signin-widget/dist/css/okta-sign-in.min.css';
```

Comment:
This import is required so the pages that need this CSS add the import in `head` when vuepress renders these.